Formula/percona-server@5.5.rb

```diff
@@ -1,10 +1,9 @@
 class PerconaServerAT55 < Formula
   desc "Drop-in MySQL replacement"
-  homepage "https://www.percona.com/"
-  url "https://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.41-37.0/source/tarball/percona-server-5.5.41-37.0.tar.gz"
-  version "5.5.41-37.0"
-  sha256 "4de65ccbdd6c266f18339c2ea5427a15d90a8ce1ce1c7574aa2e72f685a10833"
-  revision 1
+  homepage "https://www.percona.com"
```

Comment:
As a general good practice for forming URLs. If not there, f.e. the browser or curl will have to add it.